app/controllers/ConfigurationController.scala

```diff
@@ -57,18 +57,18 @@ class ConfigurationController @Inject()(val messagesApi: MessagesApi) extends Co
   }
 
   def readDataSetDefault(dataSetName: String) = SecuredAction.async { implicit request =>
-    for {
-      dataset: DataSet <- DataSetDAO.findOneBySourceName(dataSetName) ?~> Messages("dataset.notFound")
```

Comment:
What happened to the `Messages("dataset.notFound")` case?

Reply:
I tried to implement it with a nested for but couldn’t get it to compile. You are right, the error message is lost in this change (but the error itself will still come). I think that’s okay for now, as those configurations are loaded only for existing datasets anyway by the frontend (the user can’t do much with the error message anyway)